eeos/src/test/java/com/blackcompany/eeos/comment/application/model/CommentModelTest.java

```diff
@@ -0,0 +1,20 @@
+package com.blackcompany.eeos.comment.application.model;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CommentModelTest {
+
+	@Test
+	@DisplayName("내용 길이가 500자 이상일 경우 예외가 발생하지 않는다.")
+	void createTest() {
+		// given
+		String length_501_word =
```

Comment:
```suggestion
String length_501_word = new Random().ints(32, 127)
        .limit(501)
        .mapToObj(i -> String.valueOf((char) i))
        .collect(Collectors.joining());
}
```

이런식으로 랜덤 문자열을 생성해보는 건 어떨까요? 나중에 규칙이 변하더라도 쉽게 변경가능할 것 같아요~!